src/components/FieldGrid/styles.module.scss

```
@@ -0,0 +1,44 @@
+    /* Row gap floor is 12px by design decision (Frank, 2026-07-20) — the
+       default sits comfortably above it. */
+    gap: var(--field-grid-row-gap, 16px) var(--field-grid-column-gap, 20px);
+    grid-template-columns: repeat(
+        auto-fit,
+        minmax(
+            min(
+                100%,
+                max(
+                    var(--field-grid-min, 280px),
+                    calc(
+                        (100% - (var(--field-grid-max-columns, 3) - 1) * var(--field-grid-column-gap, 20px)) /
```

Stylelint (17.14.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)